src/modules/normalize/filter_audiolevel.c

```diff
@@ -61,6 +61,7 @@ static int filter_get_audio( mlt_frame frame, void **buffer, mlt_audio_format *f
 	mlt_properties filter_props = MLT_FILTER_PROPERTIES( filter );
 
 	int iec_scale = mlt_properties_get_int( filter_props, "iec_scale" );
+	int peak = mlt_properties_get_int( filter_props, "peak" );
```

Comment:
Why "peak" instead of something about dbFS unit of measure? Also, the current code is already supposed to be like a peak meter when iec_scale=1 as described [here on wikipedia](https://en.wikipedia.org/wiki/Peak_programme_meter#IEC_60268-18_Digital_PPM). Looking at the new code below, this looks like a different, non-averaging implementation of peak. Is it true peak?